javascript/javascript2/week2/event-application1.js

```diff
@@ -0,0 +1,15 @@
+function myFunction(numdays) {
```

Comment:
Use descriptive function name. See the problem description and give meaningful name.